app/src/main/java/com/sopt/instagram/ui/main/MainActivity.kt

```diff
@@ -1,12 +1,66 @@
 package com.sopt.instagram.ui.main
 
+import android.content.Intent
 import android.os.Bundle
+import android.view.Menu
+import android.view.MenuItem
+import androidx.fragment.app.Fragment
 import com.sopt.instagram.R
 import com.sopt.instagram.databinding.ActivityMainBinding
+import com.sopt.instagram.ui.home.HomeFragment
+import com.sopt.instagram.ui.newpost.NewPostActivity
 import com.sopt.instagram.util.binding.BindingActivity
 
 class MainActivity : BindingActivity<ActivityMainBinding>(R.layout.activity_main) {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+
+        initBnvItemSelectedListener()
+        initToolBar()
+    }
+
+    private fun initBnvItemSelectedListener() {
+        val currentFragment = supportFragmentManager.findFragmentById(R.id.fcv_main)
+        if (currentFragment == null) {
+            supportFragmentManager.beginTransaction().add(R.id.fcv_main, HomeFragment()).commit()
+        }
+        binding.bnvMain.setOnItemSelectedListener { item ->
+            when (item.itemId) {
+                R.id.menu_home -> {
+                    this.changeFragment(HomeFragment())
+                }
+
+                R.id.menu_new_post -> {
+                    startActivity(Intent(this, NewPostActivity::class.java))
+                }
+
+                else -> {
+                    // 아무것도 없음
+                }
+            }
+            return@setOnItemSelectedListener true
+        }
+    }
```

Comment:
```suggestion
    private fun initBnvItemSelectedListener() {
    supportFragmentManager.findFragmentById(R.id.fcv_main)
        ?: supportFragmentManager.beginTransaction().add(R.id.fcv_main, HomeFragment()).commit()
        binding.bnvMain.setOnItemSelectedListener { menu ->
            when (menu.itemId) {
                R.id.menu_new_post -> {
                    startActivity(Intent(this, NewPostActivity::class.java))
                }
                else -> {  }
            }
            return@setOnItemSelectedListener true
        }
    }
```
필수는 아니지만 요런 식으로 바꿔볼 수도 있을 것 같습니다!

Reply:
앗 develop에서 branch를 파긴 했는데 feature/#4-home을 pull 받아와서 그렇게 된 것 같습니다.. 조심하겠습니당..

Reply:
훨씬 코드가 깔끔해지는 것 같네요! 참고하겠습니다!!